Sourabh_Kanwade/html/Assignment 8/question1.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Layouts</title>
+        <style>
+            .container {
+                padding: 2rem;
+                width: 70%;
+                background-color: aqua;
+                border: solid 2px orange;
+            }
+            .img-container {
+                display: flex;
+                justify-content: space-between;
+            }
+            .img-container > img {
```

Comment:
remove >

Reply:
`>` for selecting child elements
I wanted to select child `<img>` element from `.img-container`